lua/luasnip/util/log.lua

```diff
@@ -2,14 +2,19 @@
 -- 448 = 0700
 vim.loop.fs_mkdir(vim.fn.stdpath("log"), 448)
 
+local log_location = vim.fn.stdpath("log") .. "/luasnip.log"
+
 local luasnip_log_fd = vim.loop.fs_open(
-	vim.fn.stdpath("log") .. "/luasnip.log",
+	log_location,
 	-- only append.
 	"a",
 	-- 420 = 0644
 	420)
 
-
+local logsize = vim.loop.fs_fstat(luasnip_log_fd).size
+if logsize > 50*2^20 then
+	print("Luasnip's log now takes up more than 50MiB. Consider deleting it at " .. log_location)
```

Comment:
Not sure about this, it could be a bit annoying..
Opinions?

Reply:
Maybe it could be rotated on startup? Like after 1M make a new one and rename the others?

Reply:
Interesting.. maybe one main log, one old log, and when the main log reaches some threshold, delete old, copy main to old, and clear main?
That way we shouldn't delete any important older data, and sessions running concurrent with the rotation will also continue to write their logs.
+ we don't ever consume way too much storage
(had a nvim-dap-log with like 900Mb once)